### PR TITLE
Fix many issues and add more softwares

### LINKED ISSUE #73: [SOFTWARE] Add DivineMC
**What is the name of the software?**
DivineMC https://github.com/DivineMC/DivineMC

**Describe what the server is in 1-2 sentences.**
Slightly optimized fork of Purpur

**What type of mods/plugins does the software support?**
Bukkit/Spigot/Paper

**What version does the software support?**
1.19.2-1.20.4

**Additional information**
Add any other context about the software here.

### LINKED ISSUE #75: [SOFTWARE] MCHPRS
**What is the name of the software?**
https://github.com/MCHPR/MCHPRS

**Describe what the server is in 1-2 sentences.**
A multithreaded reimplementation of the Mincraft server in Rust, made for redstone plot servers.

**What type of mods/plugins does the software support?**
None

**What version does the software support?**
1.18.2

**Additional information**
There are basic reimplementations of WorldEdit and Luckperms included

### LINKED ISSUE #76: VanillaCord is not inactive

### LINKED ISSUE #77: [SOFTWARE] Canvas
**What is the name of the software?**
https://github.com/CraftCanvasMC/Canvas

**Describe what the server is in 1-2 sentences.**
Purpur fork that tries to give plugin developers unlimted freedom when making plugins

**What type of mods/plugins does the software support?**
Bukkit/Spigot/Paper

**What version does the software support?**
1.20.2, 1.20.4

**Additional information**

### LINKED ISSUE #79: Update Minestom to 1.20.4 and remove minestom-ce
image

### PR DESCRIPTION
**Pull request by TonimatasDEV**
Fixes #77 
Fixes #76 
Fixes #75 
Fixes #73 
Fixes #79 

And add more softwares of #24 
- PaperMC-unpatch
- PaperBin
- Origami
- Lantern
- PartyCraft
- OpenSandbox
- PyMine
- Purplane
- Pukkit
- PopupMC
- Polaris
- Reflx
- Spout
- Spacecraft-GT
- Snowstorm
- Rush
- RealCraft

### CODE REVIEW

**Comment by TonimatasDEV** [comment]
@LeStegii Finished

**Comment by TonimatasDEV** [comment]
Fixes #79

File: java/PLUGINS.md [hunk 1]
```diff
@@ -178,7 +178,91 @@ This list contains Minecraft Java plugins server softwares.
...
+- **Fork:** -
+- **Description:** PyMine is a server built in Python. It has support for plugins.
+- **Plugins:** py-mine
+
+### [⬛ Purplane](https://github.com/theomega24/Purplane)
+- **Version:** 1.16.5 - 1.17.1
+- **Author:** theomega24
+- **Fork:** CraftBukkit --> Spigot --> Paper --> Pufferfish --> Purpur/AirPlane --> DivineMC
+- **Description:** A fork of purpur that adds features from Airplane.
+- **Plugins:** Bukkit, Spigot, Paper
```
Comment (LeStegii) [inline java/PLUGINS.md:222]: Fork chain got messed up here.

File: java/PLUGINS.md [hunk 2]
```diff
@@ -178,7 +178,91 @@ This list contains Minecraft Java plugins server softwares.
...
+- **Description:** A CraftBukkit CB1060 fork for Beta 1.7.3 fixing bugs and adding basic features.
+- **Plugins:** Bukkit
+
 # ❌ Inactive Development
+### [⬛ Reflx](https://github.com/Jckf/Reflx)
+- **Version:** 1.3.2
+- **Author:** Jckf
+- **Fork:** -
+- **Description:** Open source Minecraft server plugin API, and reflection based implementation.
+- **Plugins:** Reflx
```
Comment (LeStegii) [inline java/PLUGINS.md:208]: Does this fit into this category then?

File: java/PLUGINS.md [hunk 3]
```diff
@@ -178,7 +178,91 @@ This list contains Minecraft Java plugins server softwares.
...
+- **Fork:** CraftBukkit --> Spigot --> Paper --> PopupMC
+- **Description:** High performance Spigot fork that aims to fix gameplay and mechanics inconsistencies.
+- **Plugins:** Bukkit, Spigot, Paper
+
+### [⬛ PaperMC-unpatch](https://github.com/x4e/PaperBin)
+- **Version:** 1.16.5
+- **Author:** x4e
+- **Fork:** CraftBukkit --> Spigot --> Paper --> PaperMC-unpatch
+- **Description:** PaperMC-unpatch is a fork of Paper which removes a patch for sand duping.
+- **Plugins:** Bukkit, Spigot, Paper
```
Comment (LeStegii) [inline java/PLUGINS.md:243]: The link is incorrect.

File: java/PLUGINS.md [hunk 4]
```diff
@@ -178,7 +178,91 @@ This list contains Minecraft Java plugins server softwares.
...
+- **Fork:** CraftBukkit --> Spigot --> Paper --> PaperBin
+- **Description:** An experiment at improving the performance of PaperMC.
+- **Plugins:** Bukkit, Spigot, Paper
+
+### [⬛ Origami](https://github.com/Minebench/Origami)
+- **Version:** 1.14.4 - 1.16.5
+- **Author:** Minebench
+- **Fork:** CraftBukkit --> Spigot --> Paper --> Origami
+- **Description:** Custom paper fork used by Minebench.de. The fork is based on the framework used in Spottedleaf's Tuinity.
+- **Plugins:** Bukkit, Spigot, Paper
```
Comment (LeStegii) [inline java/PLUGINS.md:257]: I think we should change the description to be a bit more informative.

File: java/VANILLA.md [hunk 5]
```diff
@@ -21,7 +21,61 @@ This list contains Minecraft Java vanilla server softwares.
...
+- **Author:** MCHPR
+- **Fork:** -
+- **Description:** A multithreaded reimplementation of the Mincraft server in Rust, made for redstone plot servers.
+
 # ❌ Inactive Development
+### [⬛ Spout](https://github.com/SpoutDev/Spout)
+- **Version:** 1.5.2
+- **Author:** SpoutDev
+- **Fork:** -
+- **Description:** Open source, multithreaded, voxel game engine and platform, written in Java.
```
Comment (LeStegii) [inline java/VANILLA.md:35]: If I remember correctly, spout had bukkit support.

**Review by LeStegii — CHANGES_REQUESTED** [review]

File: java/PLUGINS.md (hunk 2 above)
Comment (TonimatasDEV) [reply java/PLUGINS.md:208]: I think yes. It has a plugin system

File: java/PLUGINS.md (hunk 3 above)
Comment (TonimatasDEV) [reply java/PLUGINS.md:243]: Fixed

File: java/PLUGINS.md (hunk 4 above)
Comment (TonimatasDEV) [reply java/PLUGINS.md:257]: Fixed.

File: java/PLUGINS.md (hunk 1 above)
Comment (TonimatasDEV) [reply java/PLUGINS.md]: Done

**Comment by TonimatasDEV** [comment]
@LeStegii

**Comment by LeStegii** [comment]
Thanks for the mention!

**Pull request merged**